lib/ansible/modules/cloud/azure/azure_rm_sql_database.py

```diff
@@ -0,0 +1,442 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sql_database
+version_added: "2.5"
+short_description: Manage an Databases.
+description:
+    - Create, update and delete an instance of Databases.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
+        description:
+            - The name of the database to be operated on (updated or created).
+        required: True
+    tags:
+        description:
+            - Resource tags.
+        required: False
+    location:
+        description:
+            - Resource location.
+        required: True
```

Comment:
Like with `azure_sql_server`, can we not require this and derive the location from the resource group if not specified?